assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,24 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId) {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
Unfortunately this is not a new functionality. data-focuses is already used by some frontends and original validatorFocus code concatenates that with hash in order to find a correct element on the page.